config/routes.rb

```diff
@@ -77,7 +77,13 @@
       resource :authorization, only: [:show] do
         resources :audio_files, except: [:new, :edit]
 
-        resources :accounts, only: [:index, :show], module: :auth do
+        resources :users, except: [:new, :edit, :create, :destroy], module: :auth do
```

Comment:
Hmm... so I'm a little hesitant to have an `auth/users` endpoint.  The point of the `auth/accounts` endpoint was that it _only_ returned accounts in your auth token.  It's a bit strange to have "all-users" here.

Trying to think what it might mean to POST to the `auth/accounts` endpoint.  I guess that _should_ create a new account (if you're allowed), and set you as the opener.  Then you could POST a new `auth/accounts/1234/memberships` with a `set_user_uri` or something.

Reply:
This is a good point. I've constrained the routes so that for now, the only thing that exists on `api/v1/authorization/users` is the `api/v1/authorization/users/:user_id/accounts` endpoint. If and when we expand account creation via CMS, that's worth another look.